addons/sourcemod/scripting/shop.sp

```diff
@@ -82,6 +86,14 @@ public APLRes AskPluginLoad2(Handle myself, bool late, char[] error, int err_max
 	MarkNativeAsOptional("PbSetBool");
 	MarkNativeAsOptional("PbSetString");
 	MarkNativeAsOptional("PbAddString");
+	
+	#if defined _SteamWorks_Included
+	// SteamWorks
+	MarkNativeAsOptional("SteamWorks_GetPublicIP");
+	MarkNativeAsOptional("SteamWorks_CreateHTTPRequest");
+	MarkNativeAsOptional("SteamWorks_SetHTTPRequestRawPostBody");
+	MarkNativeAsOptional("SteamWorks_SendHTTPRequest");
+	#endif
```

Comment:
Why? SteamWorks has defined `__ext_SteamWorks_SetNTVOptional()`, so SourceMod handle and add all SteamWorks natives automatically.